arm-compute/2016-03-30/swagger/compute.json

```diff
@@ -4307,6 +4307,11 @@
           "type": "boolean",
           "description": "Specifies whether the latest model has been applied to the virtual machine."
         },
+        "vmId": {
```

Comment:
@amarzavery My understanding was that adding this field will also be propagated to the model in REST API, so _after_ merging this request new generated client will know about it, so it will be possible to get this vmId using regular `GET`.

As of now, I just updated the generated client for my project to include `vmId` property.